samples/Mvc.FileUpload/Controllers/FileController.cs

```diff
@@ -66,9 +63,8 @@ public IActionResult FormToUpload()
                 ContentDispositionHeaderValue contentDisposition;
                 ContentDispositionHeaderValue.TryParse(section.ContentDisposition, out contentDisposition);
 
-                if (MultipartRequestHelper.HasFileContentDisposition(contentDisposition))
+                if (contentDisposition.IsFileContentDisposition())
                 {
-                    var name = HeaderUtilities.RemoveQuotes(contentDisposition.Name) ?? string.Empty;
                     var fileName = HeaderUtilities.RemoveQuotes(contentDisposition.FileName) ?? string.Empty;
```

Comment:
you didn't find a way to clean this up? you talked about creating something like IFormFile from the multipart section.